modules/services/podman-linux/builds.nix

```
@@ -0,0 +1,173 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.podman;
+
+  podman-lib = import ./podman-lib.nix { inherit lib config; };
+
+  createQuadletSource = name: buildDef:
+    let
+      buildConfig = (podman-lib.deepMerge {
+        Build = {
+          AuthFile = buildDef.authFile;
+          Environment = buildDef.environment;
+          File = buildDef.file;
+          ImageTag = ([ "homemanager/${name}" ] ++ buildDef.tags);
+          Label = (buildDef.labels // { "nix.home-manager.managed" = true; });
+          PodmanArgs = buildDef.extraPodmanArgs;
+          SetWorkingDirectory = buildDef.workingDirectory;
+          TLSVerify = buildDef.tlsVerify;
+        };
+        Install = {
+          WantedBy = (if buildDef.autoStart then [
+            "default.target"
+            "multi-user.target"
+          ] else
+            [ ]);
```

Can use `optionals`. Something like
```suggestion
          WantedBy = optionals buildDef.autoStart [
            "default.target"
            "multi-user.target"
          ];
```
ought to work.